### PR TITLE
Add new import groups of country and auto*

### PR DESCRIPTION
**Pull request by Spaceenter**

### CODE REVIEW

File: internal/store/bigtable/group.go [hunk 1]
```diff
@@ -28,10 +28,14 @@ import (
...
 	"dcbranch":   0, // Used for the latest proto branch cache
 	"branch":     0, // Used for legacy branch cache
-	"frequent":   1,
-	"ipcc":       2,
-	"biomedical": 3,
-	"disaster":   4,
+	"auto1d":     1,
+	"auto1w":     2,
+	"auto2w":     3,
+	"frequent":   4,
```
Comment (pradh) [inline internal/store/bigtable/group.go:34]: What is the new plan with frequent given autoX groups?  If it continues to be daily, should it be right after auto1d?

**Review by pradh — APPROVED** [review]
Thanks!

File: internal/store/bigtable/group.go (hunk 1 above)
Comment (Spaceenter) [reply internal/store/bigtable/group.go]: done

**Review by shifucun — APPROVED** [review]
thanks for updating the golden test!!

**Pull request merged**